scene/gui/menu_button.cpp

```diff
@@ -91,6 +88,22 @@ void MenuButton::pressed() {
 		return;
 	}
 
+	show_popup();
+}
+
+void MenuButton::gui_input(const Ref<InputEvent> &p_event) {
+	BaseButton::gui_input(p_event);
+}
```

Comment:
Is it necessary? Removing the `gui_input` override from the `MenuButton` should do the same.

Reply:
Yeah that was already there before this pr and I didn't realise! Good find.

Reply:
Fixed, it has been like this since 2017 haha!